web/src/App.tsx

```diff
@@ -83,16 +85,26 @@ export default function App(): ReactElement {
                   toastActionText={__('misc.reload')}
                 />
               )}
-              <LoadingOverlay />
-              <OnboardingModal />
-              <FAQModal />
-              <InfoModal />
-              <SettingsModal />
-              <LeftPanel />
-              <MapWrapper />
-              <TimeControllerWrapper />
-              <FeatureFlagsManager />
-              <LegendContainer />
+              <Suspense>
+                <LoadingOverlay />
+              </Suspense>
+              <Suspense>
+                <OnboardingModal />
+              </Suspense>
+              <Suspense>
+                <FAQModal />
+                <InfoModal />
+                <SettingsModal />
+              </Suspense>
+              <Suspense>
+                <LeftPanel />
+                <MapWrapper />
```

Comment:
On mobile, if people arrive via a direct link to a zone/country, only the "left panel" is shown. It might make sense not to wait on the map in that case.

Reply:
I'll take a look at that tomorrow to see how it affects non-mobile users. Otherwise that makes a lot of sense.